### ISSUE TITLE
Enable CI builds for che-incubator/configbump, including updated secrets

### ISSUE THREAD

**Issue by tolusha** [eclipse/che#21988]
### Is your task related to a problem? Please describe

Currently there is no way to release a new version of `che-incubator/configbump` since release action fails [1] with the error message:

```
Run echo "${DOCKERHUB_PASSWORD}" | docker login *** --username "***" --password-stdin
  echo "${DOCKERHUB_PASSWORD}" | docker login *** --username "***" --password-stdin
  shell: /usr/bin/bash -e {0}
  env:
    DOCKERHUB_PASSWORD: ***
Error response from daemon: Get "https://***/v[2](https://github.com/che-incubator/configbump/actions/runs/4103246296/jobs/7077201012#step:6:2)/": unauthorized: Invalid Username or Password
```
Credentials are outdated.

[1] https://github.com/che-incubator/configbump/actions/runs/4103246296

### Describe the solution you'd like

- [ ] Update quay.io and github token credentials 
- [ ] Create make-release.sh and release.yml scripts, like we have in other repos (eg., https://github.com/eclipse-che/che-machine-exec/blob/main/.github/workflows/release.yml and https://github.com/eclipse-che/che-machine-exec/blob/main/make-release.sh )
- [ ] switch che-release process from creating a branch every sprint to triggering a release - https://github.com/eclipse-che/che-release/blob/main/make-release.sh#L104 vs. https://github.com/eclipse-che/che-release/blob/main/make-release.sh#L84-L86

Stretch goals:

- [ ] replace alpine-based Dockerfile with ubi8-micro based Dockerfile
- [ ] downstream, update scripts to use new rhel8-based Dockerfile instead of rhel.Dockerfile, if it's no longer needed

### Additional context

https://github.com/eclipse/che/issues/21883

**Comment by ibuziuk** [comment]
@nickboldt could you please take a look? it is very weird to see `DOCKERHUB_PASSWORD` in the logs since I do believe we only publish images to quay.io

**Comment by nickboldt** [comment]
Note thas as part of this update, we could also switch from alpine to ubi8 as the base from which images are built.

**Comment by tolusha** [comment]
I think `ubi8` is a heavier image than `alphine`, so I am against it.

**Comment by tolusha** [comment]
Then `micro` is fine for me. ^)

**Comment by nickboldt** [comment]
@dmytro-ndp has asked for something to do for Day of Learning / H&H so I've given him this issue to work on when he has time.

**Comment by dmytro-ndp** [comment]
@nickboldt: did I understand it correctly, that you had suggested to change versioning of configbump from 0.1.0 to Eclipse Che version, like che-machine-exec?

**Comment by nickboldt** [comment]
Yeah, might as well align to the other che-incubator projects and make per-sprint releases simpler.

https://quay.io/repository/che-incubator/configbump?tab=tags
vs.
https://quay.io/repository/che-incubator/che-code?tab=tags&tag=7.62.0

**Comment by nickboldt** [comment]
Note as part of having CI builds we should verify that https://github.com/che-incubator/configbump/pull/68#pullrequestreview-1348760954 is merged and triggers a new CI build

**Comment by nickboldt** [comment]
Is this still needed?

**Comment by nickboldt** [comment]
Added to sprint plan for DTW. https://github.com/orgs/eclipse/projects/53/views/2?filterQuery=%5BDevToolsWeek%5D+&pane=issue&itemId=38362130

**Comment by nickboldt** [comment]
Ready to merge (needs codeowner approval):
* https://github.com/che-incubator/configbump/pull/98

Once merged, can update this PR and merge it too:
* https://github.com/eclipse-che/che-release/pull/80

**Comment by nickboldt** [comment]
Vastly better:

![image](https://github.com/eclipse/che/assets/227597/71cf676c-4082-48ae-a650-4cfb0fa5c34d)

However need to fix  that CVE -- see https://github.com/che-incubator/configbump/pull/100/files

**Comment by nickboldt** [comment]
When run from the 7.74.x branch (where I already pushed my changes) the job passed:

https://github.com/che-incubator/configbump/actions/runs/6226555508/job/16899641726

So the PR should be safe to merge now

**Comment by nickboldt** [comment]
Hmm. tag-release failed in https://github.com/che-incubator/configbump/actions/runs/6227605614 but passed in https://github.com/che-incubator/configbump/actions/runs/6226555508 

will assume for now this is fixed and verify further next week for the 7.75 release.

**Comment by nickboldt** [comment]
![image](https://github.com/eclipse/che/assets/227597/8ea8eeb8-d52f-4320-9143-cb05160ab3d1)

![image](https://github.com/eclipse/che/assets/227597/1e2ce762-4067-459d-b836-efa488d96ecf)

Resolving.
